### ISSUE TITLE
Spglib issue with pymatgen 3.2.7

### ISSUE THREAD

**Issue by rtrottie** [materialsproject/pymatgen#277]
I upgraded to 3.2.7 and can no longer import pymatgen, and have the following error:

File "<stdin>", line 1, in <module>
  File ".../lib/python2.7/site-packages/pymatgen/__init__.py", line 17, in <module>
    from .io.smart import read_structure, write_structure, read_mol, write_mol
  File "...0/lib/python2.7/site-packages/pymatgen/io/smart.py", line 28, in <module>
    from pymatgen.io.cif import CifParser, CifWriter
  File "...2.3.0/lib/python2.7/site-packages/pymatgen/io/cif.py", line 43, in <module>
    from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
  File ".../python2.7/site-packages/pymatgen/symmetry/analyzer.py", line 54, in <module>
    raise ImportError(msg)
ImportError: Spglib required. Please either run python setup.py install for pymatgen, or install pyspglib from spglib.

but spglib and pyspglib are both installed.  Thankfully when reverting to 3.2.6 there is no longer any issue

**Comment by shyuep** [comment]
What command did you use to upgrade?

**Comment by rtrottie** [comment]
I used: 

pip install pymatgen --upgrade --no-deps

I also tried uninstalling, upgrading pyspglib, and the reinstalling with pip.

Since I posted I have installed 3.2.7 successfully on other systems, but I am still encountering the same issue on the original computer.

**Comment by shyuep** [comment]
Hmm... as far as I know, there are no changes to spglib in 3.2.6 -> 3.2.7. It may be that there was a compilation error during the upgrade on the original computer. What OS is the original computer?

**Comment by rtrottie** [comment]
CentOS 5.4

But your advice made me think of something that I probably should have checked earlier.  Reinstalling with the --no-cache-dir option to pip fixed our issue.  Thanks for your time.

**Comment by mkhorton** [comment]
I just encountered the same `ImportError` message myself when using pymatgen 3.4.0, even though `import spglib` was working just fine. This was because the Python package `pyspglib` was renamed to `spglib` for version 1.9 of spglib. To fix, I manually downloaded the last 1.8.x release of spglib and manually installed the Python package from there, and it works fine. Not sure if it's worth adding another try/except statement to analyzer.py with `import spglib._spglib as spg` for those people who are using newer versions of spglib?
